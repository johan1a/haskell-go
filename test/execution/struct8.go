package main


type human struct {
    name string
    age int
}

func main(){
    h := (human{age: 25, name: "El Duderino"})
    x := h
    fmt.Println(x.name)
    fmt.Println(x.age)
    h.name = "test"
    fmt.Println(h.name)
    fmt.Println(x.name)
}
