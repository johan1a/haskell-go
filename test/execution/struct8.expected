El Duderino
25
test
El Duderino
